src/client/components/StudentsList.js

```diff
@@ -0,0 +1,39 @@
+import React,  { useState, useEffect } from 'react';
+import { Table } from 'semantic-ui-react';
+import axios from 'axios';
+
+const StudentsList = () => {
+  const [students, setStudents] = useState([]);
+
+  useEffect(() => {
+    const getStudentsList = async () => {
+      const { data } = await axios.get('http://localhost:3002/api/students')
```

Comment:
TBD: access base url from config

Reply:
We'll want to be able to differentiate among all students and current students.  I would anticipate this URL would return all students who ever attended RPT.

/api/students/current 
or 
/api/students/cohort/rpt23
/api/students/cohort/rpt24
...

Reply:
added routes in the server-side. Will update the client-side  once rpt-roster api supports filtering students